app/dashboard/tip_views.py

```diff
@@ -379,5 +395,7 @@ def send_tip_2(request):
         'from_handle': from_username,
         'title': 'Send Tip | Gitcoin',
         'card_desc': 'Send a tip to any github user at the click of a button.',
+        'user_json': user,
+        'username': username
```

Comment:
I wonder if we need to conditionally include these context params to prevent someone crafting a malicious URL and sending it to another user

Reply:
Sure, added!